### PR TITLE
🏗 Update `import-order-excluded.json`

### PR DESCRIPTION
**Pull request by alanorozco**
Updates unmodified extensions according to rolling window of 2 weeks.

### CODE REVIEW

**Comment by alanorozco** [comment]
@rcebulko I used this script, feel free to run it whenever.

```js
const {execFile, execSync} = require('child_process');
const {readFileSync, writeFileSync} = require('fs');

const commitsSinceDaysAgo = 14;

const prefix = 'extensions';
const listFilepath = 'extensions/import-order-excluded.json';

function daysAgo(days) {
  const now = new Date();
  now.setDate(now.getDate() - days);
  return now;
}

const yyyyMmDd = (date) =>
  date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate();

const commitsSince = (since, glob = '.') =>
  execSync(`git log --since="${since}" --pretty=format:"%H" ${glob}`, {
    encoding: 'utf8',
  })
    .trim()
    .split('\n')
    .filter(Boolean);

function update() {
  const since = yyyyMmDd(daysAgo(commitsSinceDaysAgo));
  const list = JSON.parse(readFileSync(listFilepath, 'utf8'));

  console.log(`Commits since ${since}:`);

  const kept = [];
  const removed = [];

  for (const subdir of list) {
    const dirpath = `${prefix}/${subdir}`;
    const {length} = commitsSince(since, dirpath);
    console.log(subdir, length);
    (length > 0 ? kept : removed).push(subdir);
  }

  if (kept.length === list.length) {
    console.log('Nothing to update.');
    return;
  }

  console.log('Updating:', removed);

  if (process.argv.includes('--dry-run')) {
    console.log('(Dry run, not updating.)');
    return;
  }

  writeFileSync(listFilepath, JSON.stringify(kept, null, 2));

  execFile(
    './amp.js',
    [
      'lint',
      '--fix',
      `--files=${removed
        .map((subdir) => `${prefix}/${subdir}/**/*.js`)
        .join(',')}`,
    ],
    {encoding: 'utf8'}
  ).stdout.pipe(process.stdout);
}

update();
```

**Comment by alanorozco** [comment]
@rcebulko is in charge of this migration and is out until Thursday. I'll make a new PR then.

**Comment by rcebulko** [comment]
@alanorozco No need to hold off on this on my behalf; I'm happy to see it moving along on its own without me!

File: extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js [hunk 1]
```diff
@@ -14,44 +14,55 @@
...
 import {CSS as adBadgeCSS} from '../../../build/amp-story-auto-ads-ad-badge-0.1.css';
 import {createShadowRootWithStyle} from '../../amp-story/1.0/utils';
 import {dev, devAssert, userAssert} from '../../../src/log';
-import {dict} from '#core/types/object';
-import {divertStoryAdPlacements} from '#experiments/story-ad-placements';
-import {forceExperimentBranch, getExperimentBranch} from '#experiments';
-import {getPlacementAlgo} from './algorithm-utils';
+
+
+
```
Comment (rcebulko) [inline extensions/amp-story-auto-ads/0.1/amp-story-auto-ads.js:62]: This extra whitespace can occur when a file has so many imports that the "rewriting" step gets cut off (the timeout must be relatively short). Re-running `amp lint --fix --local_changes` once or twice until it no longer changes anything should resolve this automatically

File: extensions/amp-story-auto-ads/0.1/story-ad-page.js [hunk 2]
```diff
@@ -13,50 +13,51 @@
...
+import {parseJson} from '#core/types/object/json';
+import {setStyle} from '#core/dom/style';
 import {
   AdvanceExpToTime,
   StoryAdAutoAdvance,
 } from '#experiments/story-ad-auto-advance';
+import {
+  BranchToTimeValues,
+  StoryAdSegmentExp,
+} from '#experiments/story-ad-progress-segment';
```
Comment (rcebulko) [inline extensions/amp-story-auto-ads/0.1/story-ad-page.js:37]: This looks like another file where autofixing is incomplete and needs a second pass

**Review by rcebulko — APPROVED** [review]
Approving (pending fixes mentioned) to unblock

**Review by samouri — APPROVED** [review]

**Comment by rcebulko** [comment]
@alanorozco Saw this was green and merged it, figured it would keep conflict risk lower

**Pull request merged**